### ISSUE TITLE
Training from scratch

### ISSUE THREAD

**Issue by longxvu** [hunglc007/tensorflow-yolov4-tflite#308]
I tried to overfit the model by training it with 1 image (also custom class) to see if the model works. However, the detection result is bad compared to using pretrained weight. Note that my input size is different (vertically oriented), is this a reason why the model doesn't work?
Also when training, the train and test loss on the same image is very different (test loss is about 15x higher than train loss) even though the functions to compute loss are the same.
Anyone has the same issue?

**Comment by tranleanh** [comment]
I did the same thing and got the same problem with you, bro! 
This source may only work for converting and testing from darknet weights.
